ci/ray_ci/core.tests.env.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# This script is used to setup test environment for running core tests.
+
+set -exo pipefail
+
+DL=1 ./ci/env/install-dependencies.sh
```

Comment:
just curious, what does `DL=1` mean?

Reply:
i think it's deep learning; it means to install the dl-ml-cpu-requirements.txt or something along that line as an extra.

Reply:
why core tests need dl-ml ?

Reply:
not sure, I think the test_tensorflow.py needs it; probably the only one that needs DL